src/lib/mongoBackend/mongoSubscribeContext.cpp

```diff
@@ -100,7 +100,7 @@ HttpStatusCode mongoSubscribeContext
     bool notificationDone = false;
     BSONArray conds = processConditionVector(&requestP->notifyConditionVector,
                                              requestP->entityIdVector,
-                                             requestP->attributeList, oid.str(),
+                                             requestP->attributeList, oid.toString(),
```

Comment:
Just curiosity... what is the diference between using `str()` and `toString()` ?